lib/server.go

```diff
@@ -2241,26 +2241,56 @@ func (srv *Server) _handlePeerMessages(serverMessage *ServerMessage) {
 		srv._handleMempool(serverMessage.Peer, msg)
 	case *MsgDeSoInv:
 		srv._handleInv(serverMessage.Peer, msg)
+	case *MsgDeSoValidatorVote:
+		srv._handleValidatorVote(serverMessage.Peer, msg)
+	case *MsgDeSoValidatorTimeout:
+		srv._handleValidatorTimeout(serverMessage.Peer, msg)
 	}
 }
 
 func (srv *Server) _handleFastHostStuffConsensusEvent(event *consensus.FastHotStuffEvent) {
 	// This should never happen. If the consensus message handler isn't defined, then something went
 	// wrong during the node initialization. We log it and return early to avoid panicking.
 	if srv.consensusController == nil {
-		glog.Errorf("Server._handleFastHostStuffConsensusEvent: Consensus message handler is nil")
+		glog.Errorf("Server._handleFastHostStuffConsensusEvent: Consensus controller is nil")
 		return
 	}
 
 	switch event.EventType {
 	case consensus.FastHotStuffEventTypeVote:
-		srv.consensusController.HandleFastHostStuffVote(event)
+		srv.consensusController.HandleLocalVoteEvent(event)
 	case consensus.FastHotStuffEventTypeTimeout:
-		srv.consensusController.HandleFastHostStuffTimeout(event)
+		srv.consensusController.HandleLocalTimeoutEvent(event)
 	case consensus.FastHotStuffEventTypeConstructVoteQC:
-		srv.consensusController.HandleFastHostStuffBlockProposal(event)
+		srv.consensusController.HandleLocalBlockProposalEvent(event)
 	case consensus.FastHotStuffEventTypeConstructTimeoutQC:
-		srv.consensusController.HandleFastHostStuffEmptyTimeoutBlockProposal(event)
+		srv.consensusController.HandleLocalTimeoutBlockProposalEvent(event)
+	}
+}
+
+func (srv *Server) _handleValidatorVote(pp *Peer, msg *MsgDeSoValidatorVote) {
+	// It's possible that the consensus controller hasn't been initialized. If so,
+	// we log an error and move on.
+	if srv.consensusController == nil {
```

Comment:
There's no isInitialized function on the consensus controller right?

Reply:
Correct. As long as the consensus controller is non-nil, then it knows how to gracefully handle this msg according to the initialization status of the event loop underneath